DataProcessing/BLSDataDownloader.cs

```diff
@@ -71,7 +71,7 @@ public class MyCustomDataDownloader : IDisposable
         /// </summary>
         /// <param name="destinationFolder">The folder where the data will be saved</param>
         /// <param name="apiKey">The Vendor API key</param>
-        public MyCustomDataDownloader(string destinationFolder, string apiKey = null)
+        public BLSUniverseDataDownloader(string destinationFolder, string apiKey = null)
```

Comment:
I believe the all the C# processing code/files can be deleted since python is being actually used? Same for dropbox downloader etc